apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidRoutes.kt

```diff
@@ -0,0 +1,66 @@
+package no.nav.etterlatte.trygdetid
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.application.log
+import io.ktor.server.request.receive
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.application
+import io.ktor.server.routing.get
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+
+fun Route.trygdetid(trygdetidService: TrygdetidService) {
+    route("/api/trygdetid") {
+        val logger = application.log
+
+        get {
+            logger.info("Henter trygdetid")
+            val trygdetid = trygdetidService.hentTrygdetid()
+            call.respond(TrygdetidDto.toDto(trygdetid))
+        }
+
+        post("/grunnlag") {
+            logger.info("Lagre trygdetidgrunnlag")
+            val trygdetidDto = call.receive<TrygdetidGrunnlagDto>()
+            trygdetidService.lagreTrygdetidGrunnlag(trygdetidDto.fromDto())
+            call.respond(HttpStatusCode.OK)
+        }
+    }
+}
+
+data class TrygdetidDto(
+    val grunnlag: List<TrygdetidGrunnlagDto>
+) {
+    companion object {
+        fun toDto(trygdetid: Trygdetid): TrygdetidDto {
```

Comment:
Litt personlig preferanse dette, men en mulighet er å bruke en extension-funksjon for å konvertere til og fra dto også. Da slipper man å ha konverteringen som en del av objektet.